Samples/Swift/DaysUntilBirthday/Shared/ViewModels/VerifiedAgeViewModel.swift

```diff
@@ -32,12 +41,31 @@ final class VerifiedAgeViewModel: ObservableObject {
   /// Creates an instance of this view model.
   init() {
     self.verificationState = .unverified
+    self.ageVerificationSignal = "Signal Unavailable"
+    self.isShowingAgeVerificationSignal = false
```

Comment:
I don't think this needs to be set in the initializer if you give it a default value.

Reply:
Done.